client/components/domains/mixins/domain-suggestion-mixin.jsx

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
Why introduce this as a mixin, when React is moving away from them? I don't see the benefit of it being a mixin, but maybe I'm missing something.